### PR TITLE
feat: show password feature and placeholders on sign in page

### PR DESCRIPTION
**Pull request by 17arindam**
features integrated -
-icon to show and hide password
-placeholders for username and password
Screenshot 2024-10-02 195358
Screenshot 2024-10-02 195420
Screenshot 2024-10-02 195427

### CODE REVIEW

**Comment by ChandelAnish** [comment]
PR on the dev-gssoc branch

**Comment by 17arindam** [comment]
@ChandelAnish changed the branch to dev-gssoc

File: client/login-page/index.html [hunk 1]
```diff
@@ -155,6 +133,22 @@ <h4>"Welcome to our hiring platform! Join us to discover exciting career opportu
...
+     
+      const type = password.getAttribute("type") === "password" ? "text" : "password";
+      password.setAttribute("type", type);
+
+      
+      this.classList.toggle("fa-eye-slash");
+      this.classList.toggle("fa-eye");
+    });
+
+  </script>
```
Comment (ChandelAnish) [inline client/login-page/index.html:151]: write the script in the script.js file

Comment (ChandelAnish) [inline client/login-page/index.html:151]: write this script in script.js file

**Review by ChandelAnish — COMMENTED** [review]
check the comments added to your code

**Review by ChandelAnish — COMMENTED** [review]
check the comments added to your code.
make sure you follow the suggestions

File: client/login-page/index.html (hunk 1 above)
Comment (17arindam) [reply client/login-page/index.html:151]: @ChandelAnish okay

**Comment by 17arindam** [comment]
@ChandelAnish i have corrected that please review it

**Review by ChandelAnish — APPROVED** [review]
don't make unnecessary changes like giving or removing spaces and don't remove the 
previous comments.

**Comment by 17arindam** [comment]
@ChandelAnish okayy

**Comment by ChandelAnish** [comment]
merged

**Review by ChandelAnish — COMMENTED** [review]
This issue has already been implemented. You can make the same feature for the sign up page

**Pull request merged**